src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -272,8 +272,7 @@ def generate_PCR_1_default_config(self, filename):
     pal_record_b = json.load(
         open(os.path.join('testcases', 'testdata', 'pal_record_2.json')))
 
-    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
-    # 20063955100 and 20063955200 respectively.
+    # Set the values of fipsCode in pal_records_a and pal_records_b.
```

Comment:
I suggest using something like "Use the FIPS codes of adjacent census tracts."

The current comment just reiterates what's already obvious from looking at the code, so it's not very useful.

Reply:
changed as suggested.